examples/nextjs/pages/prejoin.tsx

```diff
@@ -5,19 +5,23 @@ import { PreJoin, setLogLevel } from '@livekit/components-react';
 import type { NextPage } from 'next';
 
 const PreJoinExample: NextPage = () => {
+  const [visible, setVisible] = React.useState(true);
   setLogLevel('debug', { liveKitClientLogLevel: 'warn' });
 
   return (
     <div data-lk-theme="default" style={{ height: '100vh' }}>
-      <PreJoin
-        defaults={{ videoDeviceId: '' }}
-        onSubmit={(values) => {
-          values.audioDeviceId;
-        }}
-        onValidate={(values) => {
-          return true;
-        }}
-      />
+      <button onClick={() => setVisible((val) => !val)}>Toggle PreJoin</button>
```

Comment:
this is part of the other PR right?

Reply:
It is unrelated to the main objective of the PR, but a useful addition in general to debug the prejoin component